pkg/controllers/multiclusterservice/mcs_controller.go

```diff
@@ -119,6 +119,46 @@ func (c *MCSController) Reconcile(ctx context.Context, req controllerruntime.Req
 	return controllerruntime.Result{}, nil
 }
 
+func (c *MCSController) checkClusterConfiguration(mcs *networkingv1alpha1.MultiClusterService,
+	providerClusters, noneExistProviderClusters, consumerClusters, noneExistConsumerClusters sets.Set[string]) error {
+	if len(noneExistProviderClusters) != 0 || len(noneExistConsumerClusters) != 0 {
+		msgProvider := strings.Join(noneExistProviderClusters.UnsortedList(), ",")
+		msgConsumer := strings.Join(noneExistConsumerClusters.UnsortedList(), ",")
+		c.EventRecorder.Eventf(mcs, corev1.EventTypeNormal, events.EventReasonConfigurationRedundant,
+			fmt.Sprintf("ProviderClusters(%s)/ConsumerClusters(%s) dont's exist, Karmada will ignore them", msgProvider, msgConsumer))
+	}
+
+	if !helper.MultiClusterServiceCrossClusterEnabled(mcs) {
+		return nil
+	}
+
+	clusters := clusterv1alpha1.ClusterList{}
+	if err := c.Client.List(context.Background(), &clusters); err != nil {
+		klog.Errorf("Failed to list clusters:%v", err)
+		return err
+	}
+
+	clusterDisabledEndpointSlice := []string{}
```

Comment:
```suggestion
	var clusterDisabledEndpointSlice []string
```
Reports slice declarations with empty literal initializers used instead of nil.
An empty slice can be represented by nil or an empty slice literal. They are functionally equivalent — their len and cap are both zero — but the nil slice is the preferred style. For more information about empty slices, see Declaring Empty Slices at [github.com/golang](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices).

Reply:
Got it